lib/models/Character.js

```diff
@@ -17,6 +17,16 @@ class Character {
 
   static async getAll() {
     // implement getAll() method to return all characters with a list of quotes
+    const { rows } = await pool.query(
+      `SELECT 
+      characters.*, 
+      COALESCE(
+        json_agg(to_jsonb(quotes))
+        FILTER (WHERE quotes.id IS NOT NULL), '[]'
+    ) as quotes from characters 
+      LEFT JOIN quotes on characters.id = quotes.id
```

Comment:
just a little nudge to be joining `characters.id` on `quotes.characters_id` rather than `quotes.id` since we want to match on the same thing